.gitignore

```diff
@@ -4,6 +4,7 @@
 !vendor/**
 
 .vscode
+/.idea
```

Comment:
This shouldn't be here (same way the `.vscode` shouldn't be here), you can add this to your global gitignore

Reply:
I see IDE-specific directories very often in gitignore :shrug:

Some people keep their project settings in git for some projects, while not for others, but I guess some regex in global gitignore can solve that.

I'll remove it.